src/main/java/com/salesforce/dynamodbv2/mt/mappers/metadata/DynamoTableDescription.java

```diff
@@ -16,7 +16,7 @@
 import java.util.Optional;
 
 /**
- * Interface that allows DynamoTableDescriptionImpl's and CreateTableRequests to be used interchangeably.
+ * Interface that allows {@code DynamoTableDescriptionImpl}s and {@code CreateTableRequest}s to be used interchangeably.
```

Comment:
frankly, most of this "@code" stuff is to deal with the pluralizing of class name--not "Foo's" nor "Foos" but "{@code Foo}s" (for a hypothetical/example class Foo)